web/components/PaneMap.vue

```diff
@@ -204,41 +188,44 @@ function injectSourcesAndLayers(rawStyle, dataCountsVisible) {
   });
 
   STYLE.layers.push({
-    id: 'counts-visible-clusters',
+    id: 'counts',
+    source: 'counts',
+    'source-layer': 'counts',
     type: 'circle',
-    source: 'counts-visible',
-    filter: ['has', 'point_count'],
+    minzoom: ZOOM_MIN_COUNTS,
+    maxzoom: ZOOM_MAX + 1,
     paint: {
       'circle-color': PAINT_COLOR_COUNTS,
+      'circle-radius': 10,
       'circle-opacity': PAINT_OPACITY,
-      'circle-radius': PAINT_SIZE_COUNT_CLUSTERS,
     },
   });
 
   STYLE.layers.push({
-    id: 'counts-visible-cluster-counts',
-    type: 'symbol',
-    source: 'counts-visible',
-    filter: ['has', 'point_count'],
-    layout: {
-      'text-field': '{sum}',
-      'text-font': ['Ubuntu Regular'],
-      'text-size': 18,
-    },
+    id: 'collisions',
+    source: 'collisions',
+    'source-layer': 'collisions',
+    type: 'circle',
+    minzoom: ZOOM_MIN_COUNTS,
+    maxzoom: ZOOM_MAX + 1,
     paint: {
-      'text-color': '#1b1b1b',
+      'circle-color': '#ff0000',
+      'circle-opacity': 0.4,
+      'circle-radius': 5,
     },
   });
 
   STYLE.layers.push({
-    id: 'counts-visible-points',
+    id: 'schools',
+    source: 'schools',
+    'source-layer': 'schools',
     type: 'circle',
-    source: 'counts-visible',
-    filter: ['!', ['has', 'point_count']],
+    minzoom: ZOOM_TORONTO,
+    maxzoom: ZOOM_MAX + 1,
     paint: {
-      'circle-color': PAINT_COLOR_COUNTS,
-      'circle-opacity': PAINT_OPACITY,
-      'circle-radius': PAINT_SIZE_COUNT_POINTS,
+      'circle-color': '#777777',
```

Comment:
let's change this to a more visible colour for now (later, we'll probably want to have an icon of sorts)